### ISSUE TITLE
Поддержка кириллицы шрифтом

### ISSUE THREAD

**Issue by Y4r0z** [EtoZheSandy/L4D4_VueNuxt#3]
Текущий шрифт поддерживает только латиницу
![image](https://github.com/EtoZheSandy/L4D4_VueNuxt/assets/72458277/e2704708-c638-45b8-9603-ad917fa3fa20)

**Comment by Y4r0z** [comment]
![image](https://github.com/EtoZheSandy/L4D4_VueNuxt/assets/72458277/0908db5e-99e4-4f56-ba23-a8b14e764ff1)

**Comment by EtoZheSandy** [comment]
Верно, но это офф шрифт самой игры который очень атмосферный, аналога на ру я не нашел
